style.css

```diff
@@ -254,3 +256,172 @@ textarea:hover {
 input:hover {
   border: 1px solid #253858;
 }
+
+@media screen and (min-width: 768px) {
+  .main {
+    background: url(images/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-color: #fff;
+    border-bottom-left-radius: 100px;
+  }
+
+  body {
+    background-color: #e5e5e5;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-between;
+    padding: 40px 190px 40px 90px;
+    background-color: #fff;
+  }
+
+  .nav-links {
+    display: flex;
+  }
+
+  .nav-links ul {
+    display: flex;
+    list-style: none;
+    gap: 35px;
+  }
+
+  .nav-links ul li a {
+    color: #344563;
+    text-decoration: none;
+    font-size: 15px;
+  }
+
+  .logo {
+    font-size: 18px;
+  }
+
+  h1 {
+    font-size: 45px;
+  }
+
+  .fa-bars {
+    visibility: hidden;
+  }
+
+  .presentation {
+    font-size: 20px;
+    padding-left: 0;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .lang {
+    padding-top: 0;
+    padding-bottom: 0;
+  }
+
+  .connect {
+    font-size: 16px;
+    padding-left: 0;
+  }
+
+  .icons ul li a {
+    font-size: 20px;
+  }
+
+  .icons {
+    margin-left: 0;
+    padding: 12px;
+  }
+
+  .content {
+    width: 85%;
+  }
+
+  .works {
+    background-color: #fff;
+    display: flex;
+    width: 70%;
+    margin-bottom: 100px;
+    margin-top: 100px;
+    height: 410px;
+  }
+
+  .part {
+    height: 350px;
+  }
+
+  #section2,
+  #section4 {
+    flex-direction: row-reverse;
+  }
+
+  .body {
+    margin-left: 250px;
```

Comment:
- **Hint** It could be better to remove the margin-left to avoid the extra space to the right.